src/test/java/uk/gov/dhsc/htbhf/BrowserStackLauncher.java

```diff
@@ -19,30 +27,77 @@
 @Slf4j
 public class BrowserStackLauncher {
 
-    public static final String BROWSER_STACK_TEST_NAME = "BROWSERSTACK.TEST.NAME";
+    private static final int MAX_THREADS = 5;
+    private static ThreadLocal<String> testNameLocal = new ThreadLocal<>();
+    private static ExecutorService executorService;
+
+    private static final List<String> TEST_NAMES = List.of(
+            "mobile-android-galaxys9",
+            "windows-win10-chrome",
+            "windows-win10-firefox",
+            "windows-win10-edge",
+            "mac-mojave-chrome",
+            "mac-mojave-firefox"
+    );
 
     public static void main(String[] args) {
-        //TODO MRS 2019-08-23: Add in all the test permutations and run them in parallel with a max of 5 threads.
-        runTest("windows-win10-chrome");
-        runTest("windows-win10-firefox");
+        setRootLoggerLevel();
+        try {
+            executorService = Executors.newFixedThreadPool(MAX_THREADS);
+
+            CompletableFuture[] completableFuturesArray = TEST_NAMES.stream()
+                    .map(BrowserStackLauncher::runAndOutput)
+                    .toArray(CompletableFuture[]::new);
+            CompletableFuture<Void> combinedFuture = CompletableFuture.allOf(completableFuturesArray);
+
+            combinedFuture.get();
+        } catch (InterruptedException e) {
+            log.error("InterruptedException caught trying to run task", e);
+        } catch (ExecutionException e) {
+            log.error("ExecutionException caught trying to run task", e);
+        } finally {
+            executorService.shutdown();
+        }
+    }
+
+    //TODO MRS 2019-08-24: Seems to be ignoring the root logger level in application.properties so setting here for now.
+    private static void setRootLoggerLevel() {
+        Logger root = (Logger) LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME);
+        root.setLevel(Level.INFO);
```

Comment:
If anyone can spot why the setting in application.properties isn't doing its job, please let me know, I couldn't figure it out quickly so did this as an intermediary measure to quieten down the logs.